pkg/election/lease.go

```diff
@@ -32,6 +34,10 @@ const (
 	revokeLeaseTimeout = time.Second
 	requestTimeout     = etcdutil.DefaultRequestTimeout
 	slowRequestTime    = etcdutil.DefaultSlowRequestTime
+
+	unhealthyTimesRecordTimeout = 1 * time.Second
+	unhealthyTTLGCInterval      = 5 * time.Second
```

Comment:
I prefer using a more conservative timeout value, e.g. the lease timeout of 3 seconds, since as long as the lease keeps alive successfully inside a lease timeout, it means the etcd leader is just fine. Based on this, manually controlling the GC might be a better option so we can clear the cache each time the lease is kept alive.